README.md

```diff
@@ -53,39 +76,39 @@ Check out the [Getting Started](https://www.youtube.com/watch?v=7wyIi_cpodE&list
 
 ```bash
 # Pull the container image
-docker pull opsdroid/opsdroid:latest
+$ docker pull opsdroid/opsdroid:latest
 
 # Run the container
-docker run --rm -it -v /path/to/config_folder:/root/.config/opsdroid opsdroid/opsdroid:latest
+$ docker run --rm -it -v /path/to/config_folder:/root/.config/opsdroid opsdroid/opsdroid:latest
 ```
 
 ### Docker Service
 
 ```bash
 # Create the opsdroid config file
-docker config create OpsdroidConfig /path/to/configuration.yaml
+$ docker config create OpsdroidConfig /path/to/configuration.yaml
 
 # Create the service
-docker service create --name opsdroid --config source=OpsdroidConfig,target=/root/.config/opsdroid/configuration.yaml --mount 'type=volume,src=OpsdroidData,dst=/root/.config/opsdroid' opsdroid/opsdroid:latest
+$ docker service create --name opsdroid --config source=OpsdroidConfig,target=/root/.config/opsdroid/configuration.yaml --mount 'type=volume,src=OpsdroidData,dst=/root/.config/opsdroid' opsdroid/opsdroid:latest
 ```
 
 ### Ubuntu 16.04 LTS
 
 ```bash
 # Update apt-get
-sudo apt-get update
+$ sudo apt-get update
 
-# Install pip for python3 and locales
-sudo apt-get install python3-pip language-pack-en git
+# Install pip for Python3 and locales
```

Comment:
Originally this was `python3` as in the binary. If we are capitalizing we should also add a space and make it `Python 3` as in the proper name.

Reply:
Agreed 👍